samples/SocketsSample/Hubs/Payload.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+namespace SocketsSample.Hubs
+{
+    public struct Payload
```

Comment:
Why?

Reply:
Useful for seeing how complex objects are serialized. In this case I changed it to actually see that camelCasing worked and felt like it was reasonable to have the sample do that. If you want me to revert it, I'm OK with that.

Reply:
Yeah revert it. Or add a new method that has the payload InvokeWithComplexObject or something.